.github/FUNDING.yml

```
@@ -1 +1,2 @@
 liberapay: TeamNewPipe
+custom: ['https://newpipe.net/donate/']
```

This does the same thing ([source](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#about-funding-files)):
```suggestion
custom: 'https://newpipe.net/donate/'
```